README.md

```diff
@@ -1,2 +1,35 @@
-# design-tokens
-A place to store decisions we make.
+# Photon Design Tokens
+
+A design token is an abstraction of a visual property such as color, font, width, animation, etc. These raw values are language application agnostic and once transformed and formatted can be used on any platform.
+
+For example in CSS, design tokens are used as _variables_, though there will be conversions made for iOS and Android.
```

Comment:
Should we also mention that collections of design tokens applied to a framework will be stored in the `photon-themes` repo?

Reply:
Yeah, it makes sense! Let me figure out how to phrase it ☺️